File: EmployeePerfomanceEvaluationSystem/client_app/performance-evaluation-client/src/app/shared/models/user/reporting-manager-response-model.ts
export class ReportingManagerResponseModel {
    reportingManagerRequestId: number;
    reportedUserId: number;
    reportedUserEmail: string;
    currentReportingManagerId: number;
    currentReportingManagerEmail: string;
    newReportingManagerId: number;
    newReportingManagerEmail: string;
    status: boolean;

    static FormReportingRequestsModelArray(response: any): Array<ReportingManagerResponseModel> {

        var reportingRequestsArray: Array<ReportingManagerResponseModel> = new Array<ReportingManagerResponseModel>();
        for (var data of response.data) {

            var reportingRequest = new ReportingManagerResponseModel();
            reportingRequest.reportingManagerRequestId = data.reportingManagerRequestId;
            reportingRequest.reportedUserId = data.reportedUserId;
            reportingRequest.reportedUserEmail = data.reportedUserEmail;
            reportingRequest.currentReportingManagerId = data.currentReportingManagerId;
            reportingRequest.currentReportingManagerEmail = data.currentReportingManagerEmail;
            reportingRequest.newReportingManagerId = data.newReportingManagerId;
            reportingRequest.newReportingManagerEmail = data.newReportingManagerEmail;
            reportingRequest.status = data.status;

            reportingRequestsArray.push(reportingRequest);
        }

        return reportingRequestsArray;
    }
}

File: EmployeePerfomanceEvaluationSystem/ViewModels/Responses/User/ReportingManagerResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EmployeePerfomanceEvaluationSystem.ViewModels.Responses.User
{
    public class ReportingManagerResponseModel
    {
        public int ReportingManagerRequestId { get; set; }
        public int ReportedUserId { get; set; }
        public string ReportedUserEmail { get; set; }
        public int CurrentReportingManagerId { get; set; }
        public string CurrentReportingManagerEmail { get; set; }
        public int NewReportingManagerId { get; set; }
        public string NewReportingManagerEmail { get; set; }
        public bool Status { get; set; }
    }
}
